app/src/validators/roadmap.ts

```diff
@@ -7,13 +7,8 @@ const schema = {
   send: {
     body: Joi.object({
       activityId: activityId,
+      selectedFileIds: Joi.array().items(Joi.string()),
```

Comment:
If this is supposed to be an array of COMS guids, the validators should aim to be more precise if possible. Could do something like the following:
```ts
selectedFileIds: Joi.array().items(Joi.string().guid({ version: 'uuidv4' })),
```
Ref: https://joi.dev/api/?v=17.12.2#stringguid---aliases-uuid